src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -99,13 +99,13 @@ bool MulticopterLandDetector::update()
 bool MulticopterLandDetector::get_landed_state()
 {
 	// only trigger flight conditions if we are armed
-	if (!_arming.armed) {
-		_arming_time = 0;
-		return true;
+	//if (!_arming.armed) {
```

Comment:
Looks like some land detector work snuck in here. Can you look at the issues you had with @tumbili? I believe this is the same issue: https://github.com/PX4/Firmware/issues/4083